explainaboard/metric.py

```diff
@@ -317,6 +318,9 @@ def __init__(self, name: str, eaas_result: AsyncEaaSResult):
         super().__init__(data=None)
         self.name = name
         self.eaas_result = eaas_result
+        self._data: Optional[np.ndarray] = None
+
+        # TODO(odashi): remove this field: this is private but unused.
         self._corpus_value = None
```

Comment:
This is returned by `_get_corpus_value()` and may be useful in the future.

Reply:
I'd like to clarify the issue here:
1. `self._corpus_value` is set to `None` in the constructor (because this class does lazy evaluation of statistics, it is not initially set)
2. it is set to an appropriate value in in `_fetch_results()` (when lazy evaluation actually occurs)
3. it is returned by `get_corpus_value()`

This seems like a pretty standard way to implement lazy evaluation, but perhaps it could be documented better? I'm happy to do that, but I just wanted to clarify the issue.

Or is the issue that `get_corpus_value()` is not called anywhere else in the codebase? If that's the issue, I think that it's still reasonable to have it as an accessor, given that the EaaS client returns this value and it may be useful in the future.

Reply:
I read the impl. again and understood that `_fetch_results` calls an external invocation (EaaS) which may take long time, which is documented on the docstring but my editor didn't show the rightmost part of the sentence-_- Thanks for clarifying.
Lazy evaluation is actually achieved by flagging `not self._data` for every members. This is double-purposing a variable and introducing a boolean flag with explicit name (e.g., `_initialized`) is recommended.